CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# CHANGELOG for `fibered_mysql2`
+
+Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - Unreleased
+### Added
+- An adapter for Rails 4, 5, and 6.
+- Appraisals for Rails 4, 5, and 6.
+- TravisCI unit test pipeline.
+- Coverage reports via Coveralls.
```

Comment:
Can you update these to be complete independent statements.  We try to make sure that the bullet point stands alone in describing what it's doing, rather than forcing the reliance on the context of what section it's under.